### ISSUE TITLE
Correction required for Magic Burst multiplier calculations

### ISSUE THREAD

**Issue by Whitechaser** [DarkstarProject/darkstar#4026]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** **:** 30170707_0

**_Server Version_** **:** Revision is: Unknown

**_Source Branch_** **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**
References: https://www.bg-wiki.com/bg/Magic_Burst#Enhancing_Magic_Bursts
https://www.bg-wiki.com/bg/Mag._Burst_Bonus

function calculateMagicBurst() (line 689) in magic.lua calculates the multipler for magic bursts via (1) getting the base multiplier dependent on the skillchain count then (2) *adding* the MB bonus from BLM AM2 spells and the additional % damage bonus from the job trait and gear, when it should be *multiplying* the value in (1) to the total value in (2).

I'm still learning how to submit things (just installed git yesterday), but if anyone could just add one more variable (something like "burstbonus", initialized to 1.00) and then multiplying that to "burst" at the end before returning it, that should implement it nicely.

Basically the 1.something gotten up to line 714 should be multiplied to a 1.something obtained by adding the (caster:getMod(MOD_MAG_BURST_BONUS) / 100) and the (caster:getMerit(blmAMIIMerit[spell:getElement()]) - 1) * 0.03 for AM2, if applicable.
